This one may be legit (see it fail in two places)

```
09:18:39 --- FAIL: TestLoadDaemonConfigWithNetwork (0.00s)
09:18:39 	daemon_unix_test.go:46: assertion failed: 127.0.0.1 (string) != ?00000000000000000000ffff7f00000100000000000000000000ffff00000000 (string)
09:18:39 time="2018-06-08T09:18:39Z" level=info msg="Got signal to reload configuration, reloading from: /tmp/config-854461364"
09:18:39 FAIL

--- a/integration-cli/docker_cli_events_unix_test.go
+++ b/integration-cli/docker_cli_events_unix_test.go
@@ -436,7 +436,7 @@ func (s *DockerDaemonSuite) TestDaemonEvents(c *check.C) {
 		"(allow-nondistributable-artifacts=[",
 		" cluster-advertise=, ",
 		" cluster-store=, ",
-		" cluster-store-opts={",
+		" cluster-store-opts=",
 		" debug=true, ",
 		" default-ipc-mode=",
 		" default-runtime=",

--- a/vendor.conf
+++ b/vendor.conf
@@ -23,7 +23,7 @@ github.com/gotestyourself/gotestyourself cf3a5ab914a2efa8bc838d09f5918c1d44d029
 github.com/google/go-cmp v0.2.0
 
 github.com/RackSec/srslog 456df3a81436d29ba874f3590eeeee25d666f8a5
-github.com/imdario/mergo 0.2.1
+github.com/imdario/mergo v0.3.5
 golang.org/x/sync fd80eb99c8f653c847d294a001bdf2a3a6f768f5
 
 # buildkit

